website/static/js/comment.js

```diff
@@ -612,18 +614,21 @@ var onOpen = function(page, rootId, nodeApiUrl) {
  *      hasChildren: Node.hasChildren,
  *      threadId: undefined }
  */
-var init = function(selector, options) {
-    new CommentPane(selector, {
+var init = function(selectors, options) {
+    var cp = new CommentPane(selectors[1], {
```

Comment:
Why does this have a hard-coded index again? Can you add a code comment to clarify?